src/pages/DulResultRecords.js

```diff
@@ -64,7 +64,7 @@ class DulResultRecords extends Component {
       election: electionReview.election,
       dataUse,
       dul: electionReview.election.dataUseLetter,
-      downloadUrl: await Config.getApiUrl() + 'consent/' + electionReview.consent.consentId + '/dul',
+      downloadUrl: await Config.getApiUrl() + '/api/consent/' + electionReview.consent.consentId + '/dul',
```

Comment:
This is legacy code that will be going away at some point, but it illustrates why I like to have all url construction localized in ajax.js. When we distribute this kind of functionality across the pages, refactoring is harder.

Reply:
yeah I don't believe this download url is actually used anywhere